packages/fx-core/tests/component/driver/botAadApp/create.test.ts

```diff
@@ -184,4 +184,25 @@ describe("botAadAppCreate", async () => {
       }
     );
   });
+
+  it("should be good when reusing existing bot in env", async () => {
+    envRestore = mockedEnv({
+      [outputKeys.BOT_ID]: expectedClientId,
+      [outputKeys.SECRET_BOT_PASSWORD]: expectedSecretText,
+    });
+
+    const args: any = {
+      name: expectedDisplayName,
+    };
+
+    const result = await createBotAadAppDriver.execute(args, mockedDriverContext);
+
+    expect(result.result.isOk()).to.be.true;
+    expect(result.result.isOk() && result.result.value.get(outputKeys.BOT_ID)).to.be.equal(
+      expectedClientId
+    );
+    expect(
+      result.result.isOk() && result.result.value.get(outputKeys.SECRET_BOT_PASSWORD)
+    ).to.be.equal(expectedSecretText);
```

Comment:
You will need to call `envRestore()` after the test.

Reply:
`envRestore()` will be called in `afterEach()`.